warehouse/query-core/src/main/java/datawave/query/common/grouping/GroupingUtil.java

```diff
@@ -83,25 +84,24 @@ public GroupingInfo getGroupingInfo(Map.Entry<Key,Document> entry, Set<String> g
                                 int count = ((BigDecimal) countTypeAttribute.getType().getDelegate()).intValue();
                                 countKeyMap.put(countKey, count);
                             });
-            
-            Multimap<String,String> fieldToFieldWithContextMap = getFieldToFieldWithGroupingContextMap(entry.getValue(), expandedGroupFieldsList, fieldMap,
-                            groupFieldsSet, reverseModelMapping);
+            Set<String> groupingContextSet = new HashSet<>();
+            TreeMultimap<String,String> fieldToFieldWithContextMap = (TreeMultimap<String, String>) getFieldToFieldWithGroupingContextMap(entry.getValue(), expandedGroupFieldsList, fieldMap,
+                            groupFieldsSet, groupingContextSet, reverseModelMapping);
             log.trace("got a new fieldToFieldWithContextMap: {}", fieldToFieldWithContextMap);
             int longest = longestValueList(fieldToFieldWithContextMap);
```

Comment:
This is unused now.  Probably don't need the longestValueList method anymore either.